spring-cloud-kubernetes-controllers/spring-cloud-kubernetes-configuration-watcher/pom.xml

```diff
@@ -52,7 +52,7 @@
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-starter-contract-stub-runner</artifactId>
+			<artifactId>spring-cloud-contract-wiremock</artifactId>
```

Comment:
our build is currently failing, because of some changes in the dependencies...

Reply:
Lets just swap out this dependency for
```
<dependency>
			<groupId>com.github.tomakehurst</groupId>
			<artifactId>wiremock-jre8-standalone</artifactId>
			<scope>test</scope>
		</dependency>
```